sales/sale_amazon/apply.rst

```diff
@@ -0,0 +1,148 @@
+================================
+Apply for Amazon MWS Access Keys
+================================
+
+.. _amazon/developer-form:
+
+Submit the Amazon MWS Developer Registration and Assessment Form
+================================================================
+
+In order to synchronize your Amazon orders with Odoo, Amazon MWS access keys are required.
+They can be obtained by submitting the **Amazon MWS Developer Registration and Assessment form** to
+register as a developer. Once recognized by Amazon as a developer (i.e. you make use of an
+application connecting to MWS), you will be granted Amazon MWS access keys.
+
+First, visit the `Amazon Marketplace Web Service documentation
+<http://docs.developer.amazonservices.com/en_US/dev_guide/DG_Registering.html>`_ and follow the
+instructions to register as a developer. Take care to choose the form "I represent a seller
+organization integrating with Amazon MWS for its own selling account only.".
+
+Fill out the *Developer Registration and Assessment form* as suggested below and provide your own
+contact information in the **Developer contact information** section. In the **Business use
+information** section, select the correct region of your seller account. For the other sections,
+adapt your responses in accordance with your business case.
+
+Give a particular attention to the **Merchant Fulfilled Shipping** function. It should only be
```

Comment:
You should pay particular attention to the **Merchant Fulfilled Shipping** ~function~ section/field (?)

Reply:
Amazon calls that a function in the subtitle but it's indeed meaningless.
Let's just remove the word to avoid any confusion.